### PR TITLE
Add TestExtract for interval type

### LINKED ISSUE #5627: Implement TestExtract for INTERVAL types
See `io.prestosql.operator.scalar.timestamptz.TestExtract` for a starting point.

### PR DESCRIPTION
**Pull request by attilakreiner**
## Description

This change adds the `TestExtract` test class for testing the `interval` type.

## Additional context and related issues

Fixes #5627

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

**Comment by wendigo** [comment]
@attilakreiner have you signed CLA already?

File: core/trino-main/src/test/java/io/trino/operator/scalar/interval/TestExtract.java [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
+import static org.junit.jupiter.api.parallel.ExecutionMode.CONCURRENT;
+
+@TestInstance(PER_CLASS)
+@Execution(CONCURRENT)
+public class TestExtract
+{
+    private QueryAssertions assertions;
```
Comment (martint) [inline core/trino-main/src/test/java/io/trino/operator/scalar/interval/TestExtract.java:31]: Make this field `final` and instantiate the object here.

**Comment by martint** [comment]
@cla-bot check

**Review by martint — APPROVED** [review]

File: core/trino-main/src/test/java/io/trino/operator/scalar/interval/TestExtract.java (hunk 1 above)
Comment (attilakreiner) [reply core/trino-main/src/test/java/io/trino/operator/scalar/interval/TestExtract.java]: done

**Comment by attilakreiner** [comment]
> @attilakreiner have you signed CLA already?
@wendigo yep, done now.

**Review by wendigo — APPROVED** [review]

**Comment by wendigo** [comment]
@attilakreiner thanks! Change merged :)

**Comment by attilakreiner** [comment]
> @attilakreiner thanks! Change merged :)

Great, thanks! 😎🎉👍

**Pull request merged**